src/seagulls/examples/_main_menu_scene.py

```diff
@@ -17,6 +17,7 @@
     Surface,
     SurfaceRenderer
 )
+from seagulls.examples._game_state import GameState
```

Comment:
In this file, since you are already in the `examples` package, you can change this to `from ._game_state`

Reply:
Should be resolved.